src/config-new.ts

```diff
@@ -0,0 +1,19 @@
+import { ConfigFile } from '@salesforce/core';
+
+type HutteConfig = {
+  // userId: string;
```

Comment:
Have you made more progress on this point? I can take it ahead if you want some help